### PR TITLE
Add a new bio file for adrian977

### PR DESCRIPTION
**Pull request by Bogdandcapalau**
edewdwedewe

### CODE REVIEW

File: bios/adrian977-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# About Adrian
+
+- Lives in Bucharest, RO
+- Loves to learn new things
```
Comment (marianistor) [inline bios/adrian977-bio.md:4]: Good luck!

**Comment by zacharyrgreene1** [comment]
Bucharest eh...Hello from Texas!

**Comment by marianistor** [comment]
Nothing changed here?

**Comment by Vladimir-Hogarth** [comment]
Hello Maria

**Comment by Bogdandcapalau** [comment]
ok

**Pull request merged**